Resources/Textures/_Nuclear14/Structures/Storage/Crates/PreWarSuitcase.rsi/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": 1,
+    "license": "CC-BY-NC-SA-3.0",
+    "copyright": "converted by Marines for use in nuclear14",
+    "size": {
+      "x": 32,
+      "y": 32
+    },
+    "states": [
+      {
+        "name": "PreWarSuitcaseClosed"
```

Comment:
state names can be simplified to closed and open. This will mean if we inherit any code like from crates it will automatically work without duplicating and changing yaml.